stackrox/stackrox
# refactor(chaos): Simplify the chaos setup

## Description

Previously the deploy script used the `CHAOS_PROXY` env var to patch toxi-proxy in sensor but did not set the `ROX_CHAOS_PROFILE` env var in sensor. `CHAOS_PROXY` is ignored by sensor and instead sensor uses `ROX_CHAOS_PROFILE` to setup the proxy. This led to the chaos proxy setup incomplete.

This PR changes the deploy script to use `ROX_CHAOS_PROFILE` to setup the chaos proxy.

## Checklist

If any of these don't apply, please comment below.

## Testing Performed

- Deploy ACS with:
```bash
ROX_CHAOS_PROFILE = none
```
- Observe the a proxy gets created and sensor does not disconnect with central:
```bash
$ toxiproxy-cli list
Name			Listen		Upstream		Enabled		Toxics
======================================================================================
central			127.0.0.1:8989	central.stackrox:443	enabled		None
```

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.
